src/python/bot/tasks/corpus_pruning_task.py

```diff
@@ -338,25 +341,35 @@ def __init__(self, build_directory, context):
 
   def get_libfuzzer_flags(self):
```

Comment:
we should find a way to merge this with https://github.com/google/clusterfuzz/blob/32ce4d06bf75dabff18f009b5eacd3ae88e7fc51/src/python/bot/fuzzers/libFuzzer/fuzzer.py#L31.

The detect_leaks bit is also missing there.

Reply:
The libFuzzer's one uses all the options provided in the `.options` file, with hich value sanitization for `timeout` and `rss_limit_mb`. The pruning one only extracts certain known options, nothing else.

I'm not sure if we want to use all the options for pruning as well. I mean, it seems reasonable in general, but I recall we had some targets that set a particular value for the `-runs` option (in Chromium, due to some threading issue), I don't think we would want that to be used in the pruning, so I guess there might be some other corner cases.

OTOH it's probably fault of the fuzz target developer who added some unlucky flag to the `.options` file, so maybe we shouldn't care. Having `close_fd_mask` to be turned out during pruning would be definitely better than not having it, for an opposite example.